src/clashboard/clinical_trials_data.py

```diff
@@ -1,9 +1,16 @@
 
 
-class ClashInterface:
+class ClinicalTrialsData:
 
-    def __init__(self, db):
-        pass
+    studies = None
```

Comment:
variables declared outside `__init__` become *class variables* (think `static` in Java).   See [this page](https://syntaxdb.com/ref/python/class-variables) for further discussion.

These statements should be moved inside `__init__` as `self.studies = None`,  etc.